File: app/src/main/java/com/itn/securebrowser/BaseActivity.java
package com.itn.securebrowser;

import android.content.Context;
import android.app.Activity;
import com.itn.securebrowser.util.LocaleHelper;

public class BaseActivity extends Activity {
    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(LocaleHelper.wrap(newBase));
    }
}

File: app/src/main/java/com/itn/securebrowser/BaseActivity.kt
package com.itn.securebrowser

import android.content.Context
import androidx.activity.ComponentActivity

/**
 * All activities extend this class so the app locale is applied
 * consistently everywhere. To change the app language, update
 * LocaleHelper.APP_LOCALE and rebuild.
 */
open class BaseActivity : ComponentActivity() {
    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(LocaleHelper.wrap(newBase))
    }
}
